frontend/packages/noobaa-storage-plugin/src/components/object-data-reduction-card/object-data-reduction-card-item.tsx

```diff
@@ -35,7 +35,11 @@ export const EfficiencyItem: React.FC<EfficiencyItemProps> = React.memo(
   ({ efficiency, isLoading, error }) => {
     const infoText =
       'Efficiency ratio refers to the deduplication and compression process effectiveness.';
-    const stats: string = !_.isNil(efficiency) ? `${Number(efficiency).toFixed(1)}:1` : null;
+    let efficiencyValue = efficiency === 0 ? '1' : String(efficiency);
```

Comment:
```suggestion
    let efficiencyValue = efficiency === 0 ? '1' : String(efficiency.toFixed(1));
```

Reply:
can we do this? if not what are the other cases we are trying to handle with if condition?

Reply:
toFixed introduces floating-point to specified precision even if the values don't have any floating points. Something we are trying to avoid.
Ex: "1" => "1.0" we do not want such behaviour.

Reply:
```suggestion
    let efficiencyValue = efficiency === 0 || 1 ? '1' : String(efficiency.toFixed(1));
```

Reply:
what you think about this?

Reply:
Not the right approach. What if I get 2 or 3? IMO, the could should be as general as possible.

Reply:
I thought efficiency is a ratio of 1? Isn't it?

Reply:
2:1 ratio is a perfectly valid ratio. I guess values of probability lie in 0 to 1 range not ratios.